wpilibc/athena/src/PIDController.cpp

```diff
@@ -416,7 +422,7 @@ float PIDController::GetAvgError() const {
   {
     std::lock_guard<priority_recursive_mutex> sync(m_mutex);
     // Don't divide by zero.
-    if (m_buf.size()) avgError = m_setpoint - m_bufTotal / m_buf.size();
```

Comment:
Probably I don't remember exactly. I think it was just that all the PID tests would say they had an error of whatever we set the setpoint to, so that seems about right. Maybe resubmit this with that fixed?